src/libasr/pass/pass_list_expr.cpp

```diff
@@ -409,7 +409,7 @@ class ListExprReplacer : public ASR::BaseExprReplacer<ListExprReplacer>
         call_arg.m_value = is_end_present;
         args.push_back(al, call_arg);
 
-        std::string list_type_name = ASRUtils::get_type_code(x->m_type, true);
+        std::string list_type_name = ASRUtils::type_to_str_python(x->m_type);
```

Comment:
to be reverted. (https://github.com/lcompilers/lpython/commit/a166236d1ccc5a9d6c49c44f7e9751223de339f1)